src/components/Perfil/usuario/EquipoList.js

```diff
@@ -1,26 +1,24 @@
-import React from 'react';
-import {Equipo} from './Equipo';
-import { Container} from '@material-ui/core';
-
+import React from "react";
+import { Equipo } from "./Equipo";
+import { Container } from "@material-ui/core";
 
 export class EquipoList extends React.Component {
```

Comment:
Any reason for keeping class components here and using functional components in other files?